src/desktop/apps/auction_reaction/templates/index.jade

```diff
@@ -0,0 +1,8 @@
+extends ../../../components/main_layout/templates/redesign
+
+append locals
+  - assetPackage = 'auction_reaction'
+
+block body
+  #react-root
```

Comment:
Is this the view rendered by the [`layout`](https://github.com/artsy/force/pull/4323/files?file-filters%5B%5D=.jade&file-filters%5B%5D=.styl&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-9d2c1171b2d3fecb640db98d60802905R33) call?

Is there some magic that automatically renders the Reaction-housed registration form app? How does that work?